index.html

```diff
@@ -926,10 +914,50 @@ <h2>The <code>application/vc+jwt</code> Media Type</h2>
   </table>
 </section>
 
-<section id="conformance"></section>
+<section id="vc-ld-jwt-media-type">
+  <h2><code>application/vc+ld+jwt</code></h2>
+  <p>
+    This specification registers the <code>application/vc+ld+jwt</code> Media Type specifically for
+    identifying a <a data-cite="rfc7519#section-3">JWT</a> conforming to the Verifiable Credentials JWT format in the `typ` header.
+  </p>
+  <table>
+    <tr>
+      <td>Type name: </td>
+      <td>application</td>
+    </tr>
+    <tr>
+      <td>Subtype name: </td>
+      <td>application/vc+ld+jwt</td>
+    </tr>
+    <tr>
+      <td>Required parameters: </td>
+      <td>None</td>
+    </tr>
+    <tr>
+      <td>Encoding considerations: </td>
+      <td>
+        <code>application/vc+ld+jwt</code> values are encoded
+        as a series of base64url encoded values (some of which may be the
+        empty string) each separated from the next by a single period
+        ('.') character.
```

Comment:
This needs to refer to the JWT spec for normative guidance on what an encoding should look like.

Reply:
https://www.iana.org/assignments/media-types/application/at+jwt

^ for example, I think this language should stay but perhaps extra citation should be added.